source/Error/ConsoleHandler.ts

```diff
@@ -13,4 +13,4 @@ module Magic.Error {
 			console.log(message.toString())
 		}
 	}
-}
\ No newline at end of file
+}
```

Comment:
Annoyingly, no. I've looked around for a setting for this, but have come up empty. They will probably fix this, I hope.